apps/crn-server/test/controllers/research-outputs.test.ts

```diff
@@ -232,6 +232,53 @@ describe('ResearchOutputs controller', () => {
         );
       });
 
+      test('Should throw error when a research output with the same type and title (trimmed) already exists', async () => {
```

Comment:
I don't think you need these `(trimmed)` tests as the title and link are always trimmed. I would just have the original tests, include whitespace in the test payload and the `calledWith` values be trimmed.

Reply:
Sounds good too!

Reply:
After discussing with @peterstarling, we came to the conclusion that it might be better to keep all the tests separate. https://github.com/yldio/asap-hub/pull/3032/commits/6ba19cdeb45dd656c7f085fc49a10fcfaab68773#r1187240273
We can have a discussion together with @peterstarling when you're back 🙌